app/event/send-enrichment-error-event.js

```diff
@@ -0,0 +1,15 @@
+const raiseEvent = require('./raise-event')
+
+const sendEnrichmentErrorEvent = async (paymentRequest, error) => {
+  const correlationId = paymentRequest.correlationId
+  const event = {
+    id: correlationId,
+    name: 'payment-request-enrichment-error-event',
+    type: 'error',
+    message: error.message,
+    data: { paymentRequest }
+  }
+  await raiseEvent(event, 'error', error)
```

Comment:
error does not need to be passed to raise event as raise event does nothing with it

Reply:
removed